core/templates/domain/topic/editable-topic-backend-api.service.ts

```diff
@@ -203,7 +203,7 @@ export class EditableTopicBackendApiService {
 
   private _updateTopic(
       topicId: string,
-      topicVersion: string,
+      topicVersion: number,
```

Comment:
Are you sure about this change, because I am able to see strings are passed in version where the function is used.

Reply:
https://github.com/oppia/oppia/blob/36dd0bd3121e59861a4e2a7f38f1a7f937f84a7a/core/templates/domain/topic/TopicObjectFactory.ts#L47